### ISSUE TITLE
Heroku doesn't read DATABASE environment variable

### ISSUE THREAD

**Issue by ameistad** [Djenesis/contributr#85]
Heroku deployment error:

```
File "/app/contributr/contributr/settings/production.py", line 9, in <module>
NameError: name 'DATABASES' is not defined
```

**Comment by SanketDG** [comment]
This definitely happened after #79 got merged in.

**Comment by ameistad** [comment]
I'm working on it, I think i know what causes it. Testing with heroku now

**Comment by ameistad** [comment]
Fixed with PR #86
